pkg/cmd/server/bootstrappolicy/securitycontextconstraints.go

```diff
@@ -9,12 +9,19 @@ const (
 	SecurityContextConstraintPrivileged = "privileged"
 	// SecurityContextConstraintRestricted is used as the name for the system default restricted scc.
 	SecurityContextConstraintRestricted = "restricted"
+	// SecurityContextConstraintNonRoot is used as the name for the system default non-root scc.
+	SecurityContextConstraintNonRoot = "nonroot"
+	// SecurityContextConstraintHostMount is used as the name for the system default host mount only scc.
+	SecurityContextConstraintHostMount = "hostmount"
```

Comment:
Why is host mount useful?  This is effectively root access on all nodes, why would you still try constrain nodes?  This is effectively privileged.